CONTRIBUTION_GUIDE.md

```diff
@@ -0,0 +1,83 @@
+# Contribution Guide
+
+Thank you! If you've opened this guide, you're probably looking to help contribute to AI on EKS. We really appreciate
+you taking the time to help this project. To that end, we've written this guide to help you in your contributions so
+that the PR process goes by a lot smoother as well as to remind ourselves of all the things we need to keep in mind when
+we make changes to this repository.
+
+## General Contributions
+
+A few things to keep in mind when contributing anything to this repository:
+
+1) Open a discussion first: before committing code, please open a discussion so we can help guide.
```

Comment:
We can state this clearly `Open a GitHub issue first: Before committing code, please open a GitHub issue describing your proposed changes so we can provide guidance and avoid duplicate work.` 

We are not going to enable **Discussions** in GitHub for this repo.